EssyK/count_zeros.cs

```diff
@@ -0,0 +1,32 @@
+public static class CountZeros{
+    public static int CountZeros(int[] a)
+    {
+        int zerosCount = 0;
+
+        try
+        {
+            int i = 0, j = a.Length -1;
+
+            for (i = 0; i <= j; i++)
+            {
+                if (a[i].Equals(0))
```

Comment:
I think just this `if` block will be enough; and a small fix on the `i <= j` to `i < j`.